### ISSUE TITLE
Disable timeout for escape key

### ISSUE THREAD

**Issue by xlucn** [vicgeralds/vitetris#23]
Currently escape key will take about a second to actually take effect. I am not that familiar with curses programming, but there seems to be some options to make escape key to work without delay.

**Comment by vicgeralds** [comment]
This is not a problem if you compile without curses.

Curses programming is no fun if you ask me. In some ways it's much easier to write portable terminal apps without it.

**Comment by xlucn** [comment]
You are right. But without curses, the game won't take the whole terminal window, that's quite a trade off.

A workaround I found is to set `ESCDELAY=0 tetris`. Still have no idea how to do it in the source.

**Comment by vicgeralds** [comment]
I understand. There's `set_escdelay(0);` to do the same. (You might want to set it a bit higher though to make sure arrow keys for example are not read as ESC)
